tasks/main.yml

```diff
@@ -1,5 +1,35 @@
 ---
 
+
+- name: Show Clean Insights measurement collection disclaimer
```

Comment:
To maintain consistency, can you please move these tasks to a dedicated `consent.yml` file?